x/leverage/types/params.go

```diff
@@ -3,18 +3,23 @@ package types
 import (
 	fmt "fmt"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 	"gopkg.in/yaml.v3"
 )
 
 var _ paramtypes.ParamSet = &Params{}
 
 var (
-	KeyInterestEpoch = []byte("InterestEpoch")
+	KeyInterestEpoch                = []byte("InterestEpoch")
+	KeyCompleteLiquidationThreshold = []byte("CompleteLiquidationThreshold")
+	KeyMinimumCloseFactor           = []byte("MinimumCloseFactor")
 )
 
 var (
-	defaultInterestEpoch = int64(100)
+	defaultInterestEpoch                = int64(100)
```

Comment:
should we discuss these defaults with the team?

Reply:
We should - they've been picked somewhat arbitrarily at the moment.

Reply:
Ok, let's create an issue and discuss during our call. I've updated the agenda too.

Reply:
Issue is created #276